api.lisp

```diff
@@ -0,0 +1,77 @@
+;;;; -*- Mode: Lisp; indent-tabs-mode: nil -*-
+
+(in-package :split-sequence)
+
+(declaim (inline list-long-enough-p check-bounds check-tests))
```

Comment:
I don't think that you should even bother with inlining those functions. They are called only once so the runtime impact should be minimal.

Reply:
Sure, it doesn't seem to matter much if I do it one way or the other.

Reply:
Point is, such declaims are somewhat misleading to other developers. If something exists, it suggests that it exists for a reason. Only after inspecting code it becomes obvious that it is not the case. Therefore those declaims are missleading and therefore should be removed.

Reply:
You are correct, this `INLINE` declaration should be removed.